src/transformers/models/owlvit/configuration_owlvit.py

```diff
@@ -0,0 +1,315 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" OWL-ViT model configuration"""
+
+import copy
+import os
+from typing import Union
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+OWLVIT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "adirik/owlvit-base-patch32": "https://huggingface.co/adirik/owlvit-base-patch32/resolve/main/config.json",
+    "adirik/owlvit-base-patch16": "https://huggingface.co/adirik/owlvit-base-patch16/resolve/main/config.json",
+    "adirik/owlvit-large-patch14": "https://huggingface.co/adirik/owlvit-large-patch14/resolve/main/config.json",
+}
+
+
+class OwlViTTextConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`OwlViTModel`]. It is used to instantiate an OwlViT model according to the specified arguments, defining the model architecture. Instantiating a configuration with the defaults will yield a similar configuration to that of the OwlViT [adirik/owlvit-base-patch32](https://huggingface.co/adirik/owlvit-base-patch32) architecture.
+
+    Configuration objects inherit from [`PretrainedConfig`] and can be used to control the model outputs. Read the documentation from [`PretrainedConfig`] for more information.
```

Comment:
Make sure to run `make style` on your branch, which will fix all of those lines going over the max length of 119 we have in the repo.

Reply:
I will do that, I thought `make fixup` ran both `make style` and `make quality`